### PR TITLE
move Behaviour classes to the behaviour package and remove Fishbot class < organisation

### PR DESCRIPTION
**Pull request by spiderbiggen**

### CODE REVIEW

File: src/main/java/lasersharks/enemies/FishSpawner.java [hunk 1]
```diff
@@ -24,7 +18,7 @@
    * @param rng
    *          random number generator to use.
    */
-  FishBot generateFish(Random rng);
+  Enemy generateFish(Random rng);
```
Comment (Exclaminator) [inline src/main/java/lasersharks/enemies/FishSpawner.java]: Maybe rename this method and class aswell to spawnEnemy and generateEnemy.

File: src/main/java/lasersharks/enemies/FishFactory.java [hunk 2]
```diff
@@ -51,7 +50,7 @@
    * this value is used as the seed for the fish.
    */
   private Random fishRng;
-  
+
```
Comment (Exclaminator) [inline src/main/java/lasersharks/enemies/FishFactory.java]: This class should also be named enemyFactory

**Comment by Exclaminator** [comment]
Structure looks good, naming could use some more work.

**Comment by MichielvdBerg** [comment]
Looks good

**Pull request merged**